mysql-test/suite/rocksdb/t/skip_validate_tmp_table.test

```diff
@@ -0,0 +1,26 @@
+--source include/have_rocksdb.inc
+--source include/have_debug.inc
+
+# Write file to make mysql-test-run.pl expect the "crash", but don't restart the
+# server until it is told to
+--source include/expect_crash.inc
+
+CREATE TABLE t1 (pk int primary key) ENGINE=ROCKSDB;
+
+# Create a .frm file without a matching table
+--exec cp $MYSQLTEST_VARDIR/mysqld.$_server_id/data/test/t1.frm $MYSQLTEST_VARDIR/mysqld.$_server_id/data/test/t1#sql-test.frm
+
+# Restart the server with a .frm file exist but that table is not registered in RocksDB
+--source include/restart_mysqld.inc
+
+# This will append '#sql-test' to the end of new name
+set session debug="+d,gen_sql_table_name";
+rename table t1 to t2;
+set session debug= "-d,gen_sql_table_name";
+
+# Remove the corresponding .frm files
+--remove_files_wildcard $MYSQLTEST_VARDIR/mysqld.$_server_id/data/test *t1*.frm
+--remove_files_wildcard $MYSQLTEST_VARDIR/mysqld.$_server_id/data/test *t2*.frm
+
+# Restart the server with a table registered in RocksDB but does not have a .frm file
```

Comment:
Does this test clean up after itself?

Reply:
Actually, no, it doesn't and should. What is interesting here though it that mtr is not detecting the 'dirt'. Probably because the .frm files aren't present anymore it doesn't see the table even though it does still exist within MyRocks. Probably a better thing to do here is to move the .frm files rather than deleting them, then move them back in order to properly cleanup.

Reply:
Further digging seems that the test really does 'clean up' after itself. When MyRocks is restarted the final time with a table in its internal data dictionary that does not have a .frm file, it 'forgets' about that table, i.e. removes it from its data dictionary. If the test were to simply move the .frm away and then back, MyRocks will fail to start as it then detects that there is a .frm file but no entry in its data dictionary. So, while this is not 100% ideal, it seems to be about as close to a cleanup that can be done. The end result is that there will be 'orphaned' table data in the LSM that is never accessible.

Reply:
This points to an operational issue in MyRocks re. .frm-less table recovery we will have to provide means to resolve sooner or later. Is this tracked somewhere?

For the testcase, maybe a comment stating the end state of the instance would help here?

Reply:
Not tracked anywhere. A test case comment clarifying the lack of cleanup seems reasonable.

Reply:
A MYR/upstream issue for no means to recover or drop a table w/o FRM has to be filed, could be as brief as possible for now